src/network/FourSubstationsNodeBreakerFactory.cpp

```diff
@@ -0,0 +1,651 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <powsybl/network/FourSubstationsNodeBreakerFactory.hpp>
```

Comment:
To be discussed : I'm not sure adding this network in this PR is relevant as this network does not contain every equipments (no battery, no SVC and DL) BUT contains a BusbarSection.
With `TerminalVisitorAll` test which hase every equipments but BusbarSection code coverage we are sure that visitor traverses every equipments.

Reply:
Removed from this PR, and #276 created for that.